### PR TITLE
cli: fix changefeed update to properly handle sort-dir and start-ts

### PR DESCRIPTION
**Pull request by liuzix**
### What problem does this PR solve? 
- changefeed update does not support `--sort-dir`
- changefeed update fails silently when the user tries to update `--start-ts`

### What is changed and how it works?
- fixed the two problems above

### Check List 

Tests 

 - Unit test
 - Integration test

Related changes

 - Need to cherry-pick to the release branch
 - Need to update the documentation

### Release note

- Fix `changefeed update` to properly handle `--sort-dir` and `--start-ts`.

### CODE REVIEW

File: tests/cli/run.sh [hunk 1]
```diff
@@ -125,6 +125,18 @@ EOF
...
+    if [[ $? != 0 ]]; then
+        echo "[$(date)] <<<<< expected warning when trying to update start-ts >>>>>"
+        exit 1
+    fi
+
+    run_cdc_cli changefeed update --sort-dir "/test" --no-confirm --changefeed-id $uuid 2>&1 | grep 'deprecated'
+    if [[ $? != 0 ]]; then
+        echo "[$(date)] <<<<< expected deprecation warning when trying to update sort-dir >>>>>"
+        exit 1
+    fi
```
Comment (overvenus) [inline tests/cli/run.sh:138]: Can you check if it is updated successfully？

**Review by asddongmen — APPROVED** [review]

**Review by amyangfei — APPROVED** [review]

**Pull request closed without merging**